CMakeLists.txt

```diff
@@ -448,6 +448,7 @@ cmake_dependent_option(
   "GNU_GCC"
   OFF
 )
+# set via -DCOVERAGE=ON
 if(COVERAGE)
```

Comment:
Good catch. I think we should make this an option with a comment, that this comment is obsolete. 
If you like as a separate PR against the 2.5 branch for a quick merge.